REPOSITORIES.md

```diff
@@ -0,0 +1,152 @@
+| [cilium/tetragon](https://github.com/cilium/tetragon) | ```STABLE``` | eBPF-based Security Observability and Runtime Enforcement |
+
+## ```Ecosystem``` Scope
+
+Learn more about the ```Ecosystem``` scope in [REPOSITORY-GUIDELINES.md](https://github.com/cilium/community/blob/main/REPOSITORY-GUIDELINES.md#ecosystem).
+
+|   Name  | Status | Description |
+|:--------|:------:|:------------|
+| [cilium/arping](https://github.com/cilium/arping) | ```INCUBATING``` | Go Library to ping a host per arp datagram or query a host mac address
+| [cilium/certgen](https://github.com/cilium/certgen) | ```INCUBATING``` | A tool to generate and store certificates for Hubble Relay mTLS |
+| [cilium/chaos-testing-examples](https://github.com/cilium/chaos-testing-examples) | ```SANDBOX``` |Examples of using Cilium for chaos testing and fault injection |
+| [cilium/cilium-perf-networking](https://github.com/cilium/cilium-perf-networking) | ```SANDBOX``` | Utilities for Cilium networking performance evaluation |
+| [cilium/client-example](https://github.com/cilium/client-example) | ```SANDBOX``` | Cilium API example as linked to in the [Cilium docs](https://docs.cilium.io/en/```STABLE```/api/) |
+| [cilium/coverbee](https://github.com/cilium/coverbee) | ```SANDBOX``` |  Code coverage tooling for eBPF |
+| [cilium/fake](https://github.com/cilium/fake) | ```STABLE``` | A Go library to generate random data for testing and/or performance evaluation |
+| [cilium/health-examples](https://github.com/cilium/health-examples) | ```SANDBOX``` | Contains examples on how to use Cilium's LB health datapath |
+| [cilium/hive](https://github.com/cilium/hive) | ```INCUBATING``` | Dependency injection framework for Go |
+| [cilium/hubble-export-stdout](https://github.com/cilium/hubble-export-stdout) | ```SANDBOX``` | Export Hubble data to stdout |
+| [cilium/hubble-otel](https://github.com/cilium/hubble-otel) | ```INCUBATING``` | Hubble adaptor for OpenTelemetry |
+| [cilium/mock-hubble](https://github.com/cilium/mock-hubble) | ```SANDBOX``` | A mock implementation of Hubble API |
+| [cilium/proxy](https://github.com/cilium/proxy) | ```STABLE``` | Envoy proxy with Cilium policy enforcement filters |
+| [cilium/scaffolding](https://github.com/cilium/scaffolding) | ```INCUBATING``` | A framework for writing simple scripts to execute performance benchmarks |
+| [cilium/scruffy](https://github.com/cilium/scruffy) | ```SANDBOX``` | Utility to garbage collect docker image tags from quay.io |
+| [cilium/starwars-docker](https://github.com/cilium/starwars-docker) | ```SANDBOX``` | Deathstar as a Service |
+| [cilium/star-wars-demo](https://github.com/cilium/star-wars-demo) | ```SANDBOX``` | Star Wars themed demo including HTTP policy enforcement |
+| [cilium/star-wars-demo-automated](https://github.com/cilium/star-wars-demo-automated) | ```SANDBOX``` | Star Wars themed automated demo |
```

Comment:
maybe move these three to special? do you think they should go here?

Reply:
Yup could definitely move them there! I wasn't sure where to put them.